doc/source/start.rst

```diff
@@ -121,3 +121,24 @@ Then run ``pre-commit install`` and you're ready to go.
 .. _pre-commit: https://pre-commit.com/
 .. _have it installed: https://pre-commit.com/#install
 .. _`real git tag/revision`: https://github.com/PyCQA/bandit/releases
+
+GitLab CI/CD integration
+------------------
+
+To integrate Bandit with `GitLab continuous integration and delivery (CI/CD)`_,
+use a job like so:
+
+.. code-block:: yaml
+
+    bandit:
+        cache: {}
+        image:
+            name: cytopia/bandit:1-py3.10
```

Comment:
Thanks! Good addition. Looking at this though, makes me realise we should really start producing our own images, just in case cytopia/bandit:1-py3.10 were to fall into bad hands. I will raise an issue, but until then let's keep this PR open.

Reply:
Hello, @lukehinds! Thanks for considering this contribution. Yes, I also noticed that using the cytopia image is suboptimal and first-party Docker images would be preferable. By adding this job to the documentation, users might get encouraged to trust the source of the image. I have subscribed to the issue about creating such an image, so I can update this PR with the image name.

Reply:
We now have official images: https://github.com/PyCQA/bandit?tab=readme-ov-file#container-images